package.json

```diff
@@ -26,12 +26,8 @@
     "lab": "5.x.x"
   },
   "scripts": {
-    "test": "make test-cov"
+    "test": "lab -t 100 -L",
```

Comment:
I was wondering about that. What's the purpose of doing that since code is explicitly used in all the test cases anyway?